components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -437,4 +438,72 @@ - (void)testDynamicTypeForAttributedTextStringWhenContentSizeCategoryIsExtraExtr
   }
 }
 
+- (void)testPreferredFontForAXXLContentSizeCategory {
+  if (@available(iOS 11.0, *)) {
+    // Given
+    self.bannerView = [[MDCBannerView alloc] init];
+    [self.bannerView applyThemeWithScheme:[[MDCContainerScheme alloc] init]];
+    self.bannerView.textView.text = kBannerLongText;
+    MDCButton *button1 = self.bannerView.leadingButton;
+    [button1 setTitle:@"Action1" forState:UIControlStateNormal];
+    [button1 setTitleColor:UIColor.blackColor forState:UIControlStateNormal];
+    button1.uppercaseTitle = YES;
+    button1.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = NO;
```

Comment:
Let's chat when you get to the office. I'm surprised that this was necessary.  I'd also suggest setting `button.enableTitleFontForState = NO` so the `titleLabel.font` isn't replaced.

Reply:
Done